cloud-azure/src/main/resources/templates/arm-v2-freeipa.ftl

```diff
@@ -68,7 +68,7 @@
   	},
     "resources": [
             <#list igs as group>
-                <#if group.availabilitySetName?? && group.availabilitySetName?has_content>
+                <#if !multiAz && group.availabilitySetName?? && group.availabilitySetName?has_content>
```

Comment:
is it possible that multiAz is set to false and `instance.instance.availabilityZone` is not et?

Reply:
That will always be the case. instance.instance.availabilityZone will only be set when multiAz is true